src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -237,24 +244,76 @@ class SingleActiveFIMap extends React.Component<
     };
     const namePaths =
       plan.jurisdiction_name_path instanceof Array ? plan.jurisdiction_name_path : [];
-    const pages = namePaths.map(namePath =>
+    const pages = namePaths.map(namePath => {
       // return a page object for each name path
-      ({
+      return {
         label: namePath,
         url: '',
-      })
-    );
+      };
+    });
     breadCrumbProps.pages = [homePage, basePage, ...pages, secondLastPage];
+    const statusBadge =
+      plan && plan.plan_status === PlanStatus.ACTIVE ? (
+        <Badge color="warning" pill={true}>
+          Active
+        </Badge>
+      ) : (
+        <Badge color="success" pill={true}>
+          Complete
+        </Badge>
+      );
+    const markCompleteButton =
+      plan && plan.plan_status === PlanStatus.ACTIVE ? (
+        <Button color="primary" size="lg" block={true}>
+          Mark as Complete
+        </Button>
+      ) : null;
+
+    const cols: string[] = Object.keys(plan);
+
+    const alias: FlexObject = {
+      plan_effective_period_end: END_DATE,
+      plan_effective_period_start: START_DATE,
+    };
 
+    const container: FlexObject[] = [];
+    if (plan.plan_fi_reason === ROUTINE_PLAN) {
+      cols.forEach((column: string) => {
+        if (column === 'plan_effective_period_end' || column === 'plan_effective_period_start') {
+          container.push(
+            <span key={column} className="detailview">
+              <b>{alias[column]}:</b> &nbsp;
+              {plan && plan[column]}
+            </span>
+          );
+        }
+      });
+    } else {
+      container.push(
+        <span>
+          <b>Case Classification:</b>&nbsp;
+          {plan && plan.plan_intervention_type ? plan.plan_intervention_type : null}
+        </span>
+      );
+    }
     return (
       <div>
         <Helmet>
           <title>{`${FOCUS_INVESTIGATION}: ${plan && plan.plan_title}`}</title>
         </Helmet>
         <HeaderBreadcrumb {...breadCrumbProps} />
-        <h2 className="page-title mt-4 mb-4">
-          {FOCUS_INVESTIGATION}: {plan && plan.plan_title}
-        </h2>
+        <div>
+          <Row>
+            <Col xs="8">
+              <h2 className="page-title mt-4 mb-4">
+                {plan && plan.plan_title} {INVESTIGATION} {statusBadge}
+              </h2>
+            </Col>
+            <Col xs="4">
+              <Select placeholder="Other area Investigations" />
```

Comment:
A few things on this:
- This select will be empty.  [Here is what it needs to contain](https://github.com/onaio/reveal-frontend/issues/198#issuecomment-512242921)
- I would move this to a different component and pass the select options to it as props
- What did you consider in using **Select** vs **AsyncSelect**?